objectrocket/instances/mongodb.py

```diff
@@ -95,6 +99,21 @@ def shards(self, add_shard=False):
         return response.json()
 
     @property
+    @util.token_auto_auth
+    def new_relic_stats(self):
+        """Get stats for this instance.
+        """
+
+        if self._new_relic_stats is None:
+            self._new_relic_stats = \
+                json.loads(requests.get('{}{}'.format(self._url,
+                           'new-relic-stats'),
+                           **self._instances._default_request_kwargs).content
+                           ).get('data')
```

Comment:
This looks good. Some error handling for non `2xx` responses might be nice, but this definitely appears as though it will do the trick.

We have a fairly robust test suite and test tools already in place, so we should get some test coverage of this block as well.